docs(changelog): update changelog (#81)
cytechmobile/radicle-planning-boards: CHANGELOG.md

# Radicle Planning Boards Change Log

## **v0.2.0** - Jul 2nd 2024

### 🚀 Enhancements

> - Add support for filtering via a text field

Reviewer: We could and perhaps should say more about this and its subfeatures.

For example that it searches titles and ids, that it does partial string matching, that it highlights on the card exactly which part of it matched, the keyboard shortcut, the escape key to unfocus, and anything else I'm forgetting.

### 🧹 Chores

> - Add container packaging

Reviewer: ```suggestion
- Add docker container packaging configuration
```

Author: Done!

# Radicle Planning Boards Change Log

## **v0.2.0** - Jul 2nd 2024

### 🚀 Enhancements

- Add support for filtering via a text field
  - Uses partial string matching on the id, title, and labels of cards
  - Filters the cards while keeping their column and order

>   - Visually highlights the matched part(s) on the cards

Reviewer: ```suggestion
  - Visually highlights on each card the matched substring(s)
```

Reviewer: Nit αλλά επειδή είναι σημαντικό feature και το proposed copy το κάνει market είπα να το γράψω.

Author: Done!

  - Features shortcuts for focusing and blurring the filterbox (`Ctrl/Cmd+F` and `Escape` respectively)

>   - Syncs the filterbox value with the URL for enhanced shareability

Reviewer: ```suggestion
  - filtering state synced with and driven by the URL means that any links you copy at any time and send to others will result in the same cards shown on their screen
```

kinda on the nose but whatever :P

Author: 👍 done
